cmd/root.go

```diff
@@ -22,39 +22,48 @@ THE SOFTWARE.
 package cmd
```

Comment:
recommend updating the ReadMe, trying to understand what kind of rules are you trying to support.

Reply:
Also, can yaml be converted to json and vice versa, or input format in == output format